fe/fe-core/src/main/java/com/starrocks/connector/jdbc/JDBCConnector.java

```diff
@@ -49,7 +49,9 @@ public JDBCConnector(ConnectorContext context) {
         validate(JDBCResource.PASSWORD);
         validate(JDBCResource.DRIVER_URL);
 
-        computeDriverChecksum();
+        if (this.properties.get(JDBCResource.CHECK_SUM) == null) {
```

Comment:
Do you want to write `this.properties.get(JDBCResource.CHECK_SUM) != null`?

Reply:
@imay no, this is what I want, when we create catalog we don't need to give the check_sum, `fe` will compute it and put it into properties, this will be used to transfer it to `be` to confirm when we use it. when `fe` restart, the checksum has existed, we don't need to compute it again.

Reply:
please give it comments; it is too confusing for me.

Reply:
done